### PR TITLE
add verbose output option and reduce default logging level

### PR DESCRIPTION
**Pull request by gaurdro**

### CODE REVIEW

File: beaver/utils.py [hunk 1]
```diff
@@ -46,13 +47,13 @@ def parse_args():
     parser.add_argument('-P', '--pid', help='path to pid file', default=None, dest='pid')
     parser.add_argument('-t', '--transport', help='log transport method', dest='transport', default=None, choices=['kafka', 'mqtt', 'rabbitmq', 'redis', 'sqs', 'kinesis', 'stdout', 'tcp', 'udp', 'zmq', 'http'])
     parser.add_argument('-e', '--experimental', help='use experimental version of beaver', dest='experimental', default=False, action='store_true')
-    parser.add_argument('-v', '--version', help='output version and quit', dest='version', default=False, action='store_true')
+    parser.add_argument('-V', '--version', help='output version and quit', dest='version', default=False, action='store_true')
```
Comment (josegonzalez) [inline beaver/utils.py]: Please revert the change where flags mean different things.

**Comment by JamieCressey** [comment]
Fixes #184

**Comment by JamieCressey** [comment]
Thanks for the PR!

I'm not keen on changing the default behaviour which existing users expect. Perhaps leave the default level at INFO, with the ability to tune it down to WARN/ERROR, what are your thoughts @josegonzalez ? 

Could you also add a couple of lines to the user docs explaining the new ability, please?

Thanks again for the PR.

**Comment by josegonzalez** [comment]
Sounds good to me, @JamieCressey .

**Comment by JamieCressey** [comment]
Closing due to inactivity. Feel free to reopen!
